circuit_puzzles/programs/statutes_update_price.clsp

```diff
@@ -1,14 +1,18 @@
 (mod ((SINGLETON_MOD_HASH . SINGLETON_LAUNCHER_HASH) STATUTES PRICE_INFO PAST_CUMULATIVE_STABILITY_FEE_DF PAST_CUMULATIVE_INTEREST_DF
       CAT_MOD_HASH CRT_TAIL_HASH PRICE_UPDATE_COUNTER
       _
-      (@ mutation_value (oracle_inner_puz_hash price last_updated))
+      (@ mutation_value (oracle_inner_puz_hash price last_updated current_timestamp))
       crt_inner_puzzle_hash)
   (include *standard-cl-23*)
   (include curry.clib)
   (include condition_codes.clib)
   (include statutes_utils.clib)
   (include utils.clib)
 
+  (defun-inline price-expiration-statute statutes
+    (f (f (r (r (r (r (r (r (r (r (f statutes)))))))))))
```

Comment:
why the innermost (f ? what is (r STATUTES)?

Reply:
it's because args doesn't have paranthesis, will add it for better readability